### DISCUSSION TITLE
custom url protocol not working

### DISCUSSION THREAD

**Discussion by jadav1982** [cefsharp/CefSharp#3333] [Q&A]
I have implemented  custom url protocol using below process. 

https://stackoverflow.com/questions/80650/how-do-i-register-a-custom-url-protocol-in-windows

It's working in Brower and WebView2 but we are getting error some in WebView2 so that we are going to replace cefsharp. cefsharp is not support to custom url protocol 

we are using cefsharp 86.0.241 version.

Thanks

**Comment by amaitland** [comment]
I've added an example to https://github.com/cefsharp/CefSharp/wiki/General-Usage#protocol-execution

---

Background:

Historically you would need to implement `OnProtocolExecution` to defer to the operating system opening a custom protocol. Unfortunately there's a problem at the moment where the page ends up being blank after the link is clicked, I raised an `upstream` issue at https://bitbucket.org/chromiumembedded/cef/issues/2715/onprotocolexecution-page-goes-blank-after quite some time ago.
